projects/igniteui-angular/migrations/update-12_0_0/changes/members.json

```diff
@@ -0,0 +1,26 @@
+{
+    "$schema": "../../common/schema/members-changes.schema.json",
+    "changes": [
+        {
+            "member": "onValueChange",
+            "replaceWith": "valueChange",
+            "definedIn": [
+                "IgxSliderComponent"
+            ]
+        },
+        {
+            "member": "onValueChanged",
+            "replaceWith": "valueChanged",
+            "definedIn": [
+                "IgxSliderComponent"
+            ]
+        },
+        {
+            "member": "onProgressChanged",
```

Comment:
here we need to use 'IgxLinearProgressBarComponent' and 'IgxCircularProgressBarComponent' for describing that change. Because those are the two references that are used by the users in their applications. Which means that we should have two objects here per each class.